src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,94 @@
+package seedu.duke;
+
+import java.text.DecimalFormat;
+
+public class Ui {
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_BLACK = "\u001B[30m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    public static final String ANSI_PURPLE = "\u001B[35m";
+    public static final String ANSI_CYAN = "\u001B[36m";
+    public static final String ANSI_WHITE = "\u001B[37m";
+    public static final String ANSI_GRAY = "\u001B[90m";
+    public static final String FULL_BLOCK = "█";
+    public static final String HALF_BLOCK ="▌";
+    public static final int MAX_BAR = 16;
+    public static final int BAR_WIDTH = 10;
+    public static final String SPACE = " ";
+    public static final String EMOJI_1 = " ᕦ(ò_óˇ)";
+    public static final String FITBOT_V0 = "  ______ _ _   _           _   \n" +
+            " |  ____(_) | | |         | |  \n" +
+            " | |__   _| |_| |__   ___ | |_ \n" +
+            " |  __| | | __| '_ \\ / _ \\| __|\n" +
+            " | |    | | |_| |_) | (_) | |_ \n" +
+            " |_|    |_|\\__|_.__/ \\___/ \\__|";
```

Comment:
nice design!

Reply:
The design and logo are very cool for current version already! Perhaps in the future you can consider using '\t' to indent all the messages and diagram displayed :) 
Since our application is CLI-based, so instead of focusing on how cool the output, we can focus more on how neat and easily-visible for the messages we want to show. 
Keep up the good work, good job Yi Zhi! :D